apps/trading-e2e/src/integration/home-node-network.cy.ts

```diff
@@ -0,0 +1,111 @@
+import { closeWelcomeDialog } from '../support/helpers';
+
+const dialogContent = 'dialog-content';
+const nodeHealth = 'node-health';
+
+describe('home', { tags: '@regression' }, () => {
+  before(() => {
+    cy.clearLocalStorage();
+    cy.mockTradingPage();
+    cy.mockSubscription();
+    cy.visit('/');
+  });
+
+  describe('footer', () => {
+    it('shows current block height', () => {
+      closeWelcomeDialog();
+      // 0006-NETW-004
+      // 0006-NETW-005
+      // 0006-NETW-008
+      // 0006-NETW-009
+      // 0006-NETW-011
+
+      cy.intercept('POST', 'http://localhost:3008/graphql', (req) => {
+        req.on('response', (res) => {
+          res.setDelay(3000);
+        });
+      });
+
+      cy.getByTestId(nodeHealth)
+        .children()
+        .first()
+        .should('contain.text', 'Warning delay ( >3 sec)');
+
+      cy.intercept('POST', 'http://localhost:3008/graphql', (req) => {
+        req.on('response', (res) => {
+          res.setDelay(1);
+        });
+      });
+
+      cy.getByTestId(nodeHealth)
+        .children()
+        .first()
+        .should('contain.text', 'Operational', { timeout: 10000 })
+        .next()
+        .should('contain.text', new URL(Cypress.env('VEGA_URL')).origin)
+        .next()
+        .should('contain.text', '100'); // all mocked queries have x-block-height header set to 100
+    });
+
+    it('shows node switcher details', () => {
+      // 0006-NETW-012
+      // 0006-NETW-013
+      // 0006-NETW-014
+      // 0006-NETW-015
+      // 0006-NETW-016
+
+      cy.getByTestId(nodeHealth).click();
+      cy.getByTestId(dialogContent).should('contain.text', 'Connected node');
+      cy.getByTestId(dialogContent).should(
+        'contain.text',
+        'This app will only work on CUSTOM. Select a node to connect to.'
+      );
+      cy.getByTestId('node')
+        .first()
+        .should('contain.text', new URL(Cypress.env('VEGA_URL')).origin)
+        .next()
+        .should('contain.text', 'Response time')
+        .next()
+        .should('contain.text', 'Block')
+        .next()
+        .should('contain.text', 'Subscription');
+      cy.getByTestId('custom-row').should('contain.text', 'Other');
+      cy.getByTestId('dialog-close').click();
+    });
+
+    it('switch to other node', () => {
+      // 0006-NETW-017
+      // 0006-NETW-018
+      // 0006-NETW-019
+      // 0006-NETW-020
+
+      cy.getByTestId(nodeHealth).click();
+      cy.getByTestId('connect').should('be.disabled');
+      cy.getByTestId('node-url-custom').click();
+      cy.getByTestId('connect').should('be.disabled');
+      cy.get("input[placeholder='https://']")
+        .focus()
+        .type(new URL(Cypress.env('VEGA_URL')).origin + '/graphql');
+      cy.getByTestId('connect').click();
+      cy.getByTestId(nodeHealth)
+        .children()
+        .first()
+        .should('contain.text', 'Operational');
```

Comment:
Should there be another assert to ensure the title of the node here is the same as the one clicked on in the row?